datafusion/src/logical_plan/plan.rs

```diff
@@ -289,7 +303,7 @@ impl LogicalPlan {
             LogicalPlan::Projection { schema, .. } => schema,
             LogicalPlan::Filter { input, .. } => input.schema(),
             LogicalPlan::Window { schema, .. } => schema,
-            LogicalPlan::Aggregate { schema, .. } => schema,
+            LogicalPlan::Aggregate(aggregate) => &aggregate.schema,
```

Comment:
@alamb I have question about the code style, before refactor, we use the style
```
::Aggregate {filed1, filed2, field3,....}
```
Someone may use the style after refactor

```
Aggregate(agg) => 
agg.field1
agg.field2
agg.field3
```
How about make consistent with before? 
In My pull requests #1325  #1322 #1311, I keep consistent and use [this](https://github.com/liukun4515/arrow-datafusion/blob/451a00cbe05ec385431eba97ba5a6d7a827fb982/ballista/rust/core/src/serde/logical_plan/to_proto.rs#L1000) style.